code/game/objects/items/weapons/shields.dm

```diff
@@ -103,24 +201,25 @@
 	slot_flags = SLOT_BACK
 	force = 18
 	throwforce = 8
-	base_block_chance = 60
 	throw_speed = 10
 	throw_range = 20
 	w_class = WEIGHT_CLASS_BULKY
 	origin_tech = list(TECH_MATERIAL = 1)
 	matter = list(DEFAULT_WALL_MATERIAL = 1000, MATERIAL_WOOD = 1000)
+	material_repair_type = MATERIAL_WOOD
 	attack_verb = list("shoved", "bashed")
 
+	armor = list(
+		melee = ARMOR_MELEE_KNIVES,
+		bullet = ARMOR_BALLISTIC_MINOR,
+		laser = ARMOR_LASER_MINOR,
+		energy = ARMOR_ENERGY_MINOR
+	)
+
 /obj/item/shield/buckler/handle_shield(mob/user)
 	. = ..()
-	if(.) playsound(user.loc, 'sound/weapons/Genhit.ogg', 50, 1)
-
-/obj/item/shield/buckler/get_block_chance(mob/user, var/damage, atom/damage_source = null, mob/attacker = null)
-	if(istype(damage_source, /obj/projectile))
-		var/obj/projectile/P = damage_source
-		if((is_sharp(P) && damage > 10) || istype(P, /obj/projectile/beam))
-			return 0
-	return base_block_chance
+	if(.)
+		playsound(user.loc, 'sound/weapons/Genhit.ogg', 50, 1)
```

Comment:
```suggestion
		playsound(get_turf(user), 'sound/weapons/Genhit.ogg', 50, 1)
```
Extrarange needs to be one of the defines, or based on one of the defines, no raw numbers